src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -189,6 +226,10 @@ private static void checkIsScreenDevice(GraphicsDevice device) {
 
     /**
      * Moves mouse pointer to given screen coordinates.
+     * @implNote the mouse pointer may not visually move on Linux systems
+     *           using Wayland, while the subsequent mousePress and mouseRelease
+     *           can be delivered to the correct location
```

Comment:
Updated.
Probably this can be covered by the line 80:
`<li> ignoring or limiting specific requests for Robot generated (synthesized) events related to keyboard and mouse etc.</li>`